Gargoyle uses smpeg for MP3 support, which is an unofficial extension to
the blorb spec, which governs embedded resources.

Although smpeg is provided by smpeg-libs which lives in RPM Fusion, we
still cannot use it because the Fedora version of SDL_sound is compiled
without support for smpeg (as well as without its more limited built-in MP3
support). So we end up without MP3 support. This is not a big deal since
very few games use sound at all.

Index: gargoyle/src/Jamrules
===================================================================
--- gargoyle.orig/src/Jamrules
+++ gargoyle/src/Jamrules
@@ -65,7 +65,7 @@ switch $(OS)
         if $(USESDL) = yes
         {
             GARGLKCCFLAGS += -I/usr/include/SDL ;
-            GARGLKLIBS += -lSDL_mixer -lSDL_sound -lSDL -lsmpeg -lvorbisfile ;
+            GARGLKLIBS += -lSDL_mixer -lSDL_sound -lSDL -lvorbisfile ;
         }
 
         if $(STATIC) { LINKLIBS += $(GARGLKLIBS) ; }
